x/bounty/keeper/program.go

```diff
@@ -39,3 +39,26 @@ func (k Keeper) SetNextProgramID(ctx sdk.Context, id uint64) {
 	binary.LittleEndian.PutUint64(bz, id)
 	store.Set(types.GetNextProgramIDKey(), bz)
 }
+
+func (k Keeper) TerminateProgram(ctx sdk.Context, caller sdk.AccAddress, id uint64) error {
+	program, found := k.GetProgram(ctx, id)
+	if !found {
+		return types.ErrProgramNotExists
+	}
+	host, err := sdk.AccAddressFromBech32(program.CreatorAddress)
+	if err != nil {
+		return types.ErrProgramCreatorInvalid
+	}
+	if !caller.Equals(host) {
+		return types.ErrProgramNotCreator
+	}
```

Comment:
Maybe we should also allow certifier to end any rogue programs during the early stages

Reply:
That sound good to me. done with the change.